README.md

```diff
@@ -256,6 +258,29 @@ flow.status
 
 ## Advanced features
 
+### Global parameters for jobs
+
+Workflows can accept a hash of `globals` that are automatically forwarded as parameters to all jobs.
+
+This is useful to have common functionality across workflow and job classes, such as tracking the creator id for all instances:
+
+```ruby
+class SimpleWorkflow < Gush::Workflow
+  def configure(url_to_fetch_from)
+    run DownloadJob, params: { url: url_to_fetch_from }
+  end
+end
+
+flow = SimpleWorkflow.create('http://foo.com', globals: { creator_id: 123 })
+flow.globals
+=> {:creator_id=>123}
+flow.jobs.first.params
+=> {:creator_id=>123, :url=>"http://foo.com"}
+```
+
+**Note:** job params with the same key as globals will overwrite the globals.
```

Comment:
Are there any other downsides that should be documented? E.g. can a job overwrite a global parameter or are global parameters immutable?

Reply:
Not that I can think of. I rewrote this line to clarify that the globals are not affected:

> **Note:** job params with the same key as globals will take precedence over the globals.